reactive/kubernetes_master.py

```diff
@@ -1801,6 +1844,9 @@ def configure_apiserver():
         # No point in trying to start master services and fail. Just return.
         return
 
+    # Update unit db service-cidr
+    freeze_service_cidr()
```

Comment:
```suggestion
    was_service_cidr_expanded = is_service_cidr_expansion()
    if was_service_cidr_expanded:
        # Update unit db service-cidr
        freeze_service_cidr()
```

This has to be cached or we won't be able to tell below whether there was an actual change.

Reply:
Per @Cynerva's comment elsewhere, this should instead be:

```suggestion
    was_service_cidr_expanded = is_service_cidr_expansion()
    freeze_service_cidr()
```